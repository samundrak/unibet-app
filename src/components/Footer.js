import React from 'react';

const Footer = () => {
  return (
    <div className="inner">
      <p>&copy; 1997-2015, Unibet. All rights reserved.</p>
    </div>
  );
};
export default Footer;
