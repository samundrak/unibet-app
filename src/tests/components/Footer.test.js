import React from 'react';
import { render } from 'react-dom';
import Footer from '../../components/Footer';

describe('test Footer', () => {
  it('should test snapshot', () => {
    const container = document.createElement('div');
    render(<Footer />, container);
    expect(container.innerHTML).toMatchSnapshot();
  });
});
