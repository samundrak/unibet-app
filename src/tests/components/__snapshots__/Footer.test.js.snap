// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Footer should test snapshot 1`] = `"<div class=\\"inner\\"><p>Â© 1997-2015, Unibet. All rights reserved.</p></div>"`;
